examples/tests/valid/json_bucket.w

```diff
@@ -0,0 +1,29 @@
+bring cloud;
+
+let b = new cloud.Bucket();
+let file_name = "file.json";
+
+let j = Json {
+  persons: [
+    {
+      age: 30,
+      name: "hasan",
+      fears: [
+        "heights",
+        "failure"
+      ]
+    }
+  ]
+};
+
+
+let get_json = new cloud.Function(inflight(msg:str): str => {
+  let x = b.get_json(file_name);
+  assert("failure" == x.get("persons").get_at(0).get("fears").get_at(1));
```

Comment:
How does the type checker know that this is a string?

Reply:
It doesnt really, its because right now everything is a subtype of Json which is probably not ideal. It was just a quick way to get implicit casting from native to Json. 
See: https://github.com/winglang/wing/issues/1717 

I talked with @yoav-steinberg about this earlier as he pointed it out and looking to start some conversation in the above issue.